graph-db/serverless.yml

```diff
@@ -0,0 +1,22 @@
+service: graph-db
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+  region: us-east-1
+  stage: ${opt:stage, ''}
```

Comment:
Defaulting to an empty value seems redundant, removing the default would seem to do the same thing. Possibly default to something like 'dev'?